eclipse/sw360
# feat(ProjectUI): Unified view of Linked Project and Releases, Clearing status, Linked Release Hierarchy. View includes tree view and list view

> Unified view of Linked Project and Releases, Clearing status, Linked Release Hierarchy. View includes tree view and list view
> * Which issue is this pull request belonging to and how is it solving it? (#118 )
> * Did you add or update any new dependencies that are required for your change? - No

### How To Test?
> Open Project in Details view. Navigate to `Clearing Status` tab. Information of all Linked Projects, releases and corresponding Clearing status should be available in `Tree view` and `List view`
> No additional tests implemented.

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
